### PR TITLE
Adds working version of blinky

### PR DESCRIPTION
**Pull request by johnnyman727**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,6 +4,9 @@ name = "rust_tessel"
 version = "0.0.1"
 authors = ["Kenneth Nierenhausen <kennethnierenhausen@gmail.com>"]
 
-[dependencies]
+#[dependencies]
 
-unix_socket = "0.3.0"
+# unix_socket = "0.5.0"
```
Comment (johnnyman727) [inline Cargo.toml]: Just temporary. `0.5.0` doesn't want to cross compile.

File: Cargo.toml [hunk 2]
```diff
@@ -5,5 +5,7 @@ version = "0.0.1"
 authors = ["Kenneth Nierenhausen <kennethnierenhausen@gmail.com>"]
 
 [dependencies]
+unix_socket = "0.5.0"
 
-unix_socket = "0.3.0"
+[target.mipsel-unknown-linux-gnu]
```
Comment (kevinmehall) [inline Cargo.toml]: I don't think this is necessary anymore. Your target is `tessel2` with the target JSON file, not `mipsel-unknown-linux-gnu`.

File: src/lib.rs [hunk 3]
```diff
@@ -1,145 +1,169 @@
...
+// Prefix and suffix to every LED file path
+const LED_PATH_PREFIX: &'static str = "/sys/devices/leds/leds/tessel:";
+const LED_PATH_SUFFIX: &'static str = "/brightness";
+impl LED {
+    // Factory method for creating new LEDs
+    pub fn new(color: &'static str, kind: &'static str) -> LED {
+
+        // Assemble the file path
+        let name = color.to_string() + ":" + kind;
+        let path = LED_PATH_PREFIX.to_string() + &name + LED_PATH_SUFFIX;
```
Comment (kevinmehall) [inline src/lib.rs]: `format!("/sys/device/leds/leds/tessel:{}:{}/brightness", color, kind);` could be easier.

**Comment by flaki** [comment]
Woo-hoo! :raised_hands:

**Pull request merged**